vng_api_common/constants.py

```diff
@@ -81,9 +83,27 @@ class RolTypes(DjangoChoices):
     medewerker = ChoiceItem("medewerker", "Medewerker")
 
 
+BESLUIT_CONST = "besluit"
+BESLUIT_CHOICE = ChoiceItem(BESLUIT_CONST, _("Besluit"))
+
+ZAAK_CONST = "zaak"
+ZAAK_CHOICE = ChoiceItem(ZAAK_CONST, _("Zaak"))
+
+VERZOEK_CONST = "verzoek"
+VERZOEK_CHOICE = ChoiceItem(VERZOEK_CONST, _("Verzoek"))
+
+
 class ObjectTypes(DjangoChoices):
-    besluit = ChoiceItem("besluit", _("Besluit"))
-    zaak = ChoiceItem("zaak", _("Zaak"))
+    besluit = BESLUIT_CHOICE
+    zaak = ZAAK_CHOICE
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Don't think this ever runs because DjangoChoices is all class methods, but eh, worksforme :-)